README.md

```diff
@@ -753,7 +753,8 @@ targets:
 ### NuGet (`nuget`)
 
 Uploads packages to [NuGet](https://www.nuget.org/) via [.NET Core](https://github.com/dotnet/core).
-By default, `craft` publishes all packages with `.nupkg` extension.
+By default, `craft` publishes all packages with `.nupkg` extension by using `dotnet nuget publish --skip-duplicate`
+so re-running `craft` means it will not fail, but instead continue to publish any package it failed.
```

Comment:
@BYK Or how about instead?

```suggestion
By default, `craft` publishes all packages with `.nupkg` extension. The target is re-entrant because of `--skip-duplicate`, which allows working around flakiness on the NuGet API by re-running the `craft`.
```